install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -1,4 +1,6 @@
-SET SESSION sql_mode='';
+SET SESSION sql_mode = '';
 SET NAMES 'utf8';
 
 ALTER TABLE `PREFIX_search_word` MODIFY `word` VARCHAR(30) NOT NULL;
+
+/* PHP:ps_1770_preset_tab_enabled(); */;
```

Comment:
Shouldn't this method **ps_1770_preset_tab_enabled** be called outside of a sql file?  (or is it somehow parsed and executed?)

Reply:
It's parsed and executed https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Install/Upgrade.php#L456